src/ResourceManager/EventHub/Commands.EventHub/Cmdlets/Namespace/NewAzureEventHubNamespace.cs

```diff
@@ -24,142 +24,67 @@ namespace Microsoft.Azure.Commands.EventHub.Commands.Namespace
         /// </summary>
-        [Parameter(
-            Mandatory = true,
-            ValueFromPipelineByPropertyName = true,
-            Position = 2,
-            HelpMessage = "EventHub Namespace Location.",
-            ParameterSetName = NamespaceParameterSet
-            )]        
-        [Parameter(
-            Mandatory = true,
-            ValueFromPipelineByPropertyName = true,
-            Position = 2,
-            HelpMessage = "EventHub Namespace Location.",
-            ParameterSetName = AutoInflateParameterSet
-            )]
+        [Parameter(Mandatory = true, ParameterSetName = NamespaceParameterSet, ValueFromPipelineByPropertyName = true, Position = 2, HelpMessage = "EventHub Namespace Location.")]        
+        [Parameter(Mandatory = true, ParameterSetName = AutoInflateParameterSet, ValueFromPipelineByPropertyName = true, Position = 2, HelpMessage = "EventHub Namespace Location.")]
         [LocationCompleter("Microsoft.EventHub/namespaces")]
         [ValidateNotNullOrEmpty]
         public string Location { get; set; }
 
         /// <summary>
         /// Namespace Sku Name.
         /// </summary>
-        [Parameter(
-          Mandatory = false,
-            Position = 3,
-          ValueFromPipelineByPropertyName = true,
-          HelpMessage = "Namespace Sku Name.",
-          ParameterSetName = NamespaceParameterSet
-            )]
-        [ValidateSet(SKU.Basic,
-          SKU.Standard,
-          SKU.Premium,
-          IgnoreCase = true)]
-        [Parameter(
-          Mandatory = false,
-            Position = 3,
-          ValueFromPipelineByPropertyName = true,
-          HelpMessage = "Namespace Sku Name.",
-          ParameterSetName = AutoInflateParameterSet
-            )]
+        [Parameter(Mandatory = false, ParameterSetName = NamespaceParameterSet, ValueFromPipelineByPropertyName = true, Position = 3, HelpMessage = "Namespace Sku Name.")]        
+        [Parameter(Mandatory = false, ParameterSetName = AutoInflateParameterSet, ValueFromPipelineByPropertyName = true, Position = 3, HelpMessage = "Namespace Sku Name.")]
+        [ValidateSet(SKU.Basic, SKU.Standard, SKU.Premium, IgnoreCase = true)]
         public string SkuName { get; set; }
 
         /// <summary>
         /// The eventhub throughput units.
         /// </summary>
-        [Parameter(
-          Mandatory = false,
-            Position = 4,
-          ValueFromPipelineByPropertyName = true,
-          HelpMessage = "The eventhub throughput units.",
-          ParameterSetName = NamespaceParameterSet
-            )]
-        [Parameter(Mandatory = false,
-            Position = 4,
-          ValueFromPipelineByPropertyName = true,
-          HelpMessage = "The eventhub throughput units.",
-          ParameterSetName = AutoInflateParameterSet)]
+        [Parameter(Mandatory = false, ParameterSetName = NamespaceParameterSet, ValueFromPipelineByPropertyName = true, HelpMessage = "The eventhub throughput units.")]
```

Comment:
Add position back for both parameter attributes.

Reply:
added back